modules/proc_deploy-quay-openshift-operator-tng.adoc

```diff
@@ -285,6 +285,14 @@ Type "help" for help.
 postgres=# \password
 ```
 
+To create a `Secret` containing the `postgres` root admin users' password:
+```sh
+$ oc create secret generic quaydb --from-literal=database-root-password=postgresql 
```

Comment:
Copy the command from [this section of the old docs](https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/deploy_red_hat_quay_on_openshift_with_quay_operator/index#specifying_database_credentials).